Podcast/UserSettings.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  UserSettings.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/23/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class UserSettings: NSObject {
+    
+    static let sharedSettings = UserSettings()
+    
+    private var saveCounter: Int!
+    private let maxSave: Int = 5
+    
+    var pushNotifications: Bool!
+    
+    private override init() {
+        saveCounter = 0
```

Comment:
What is this for? I don't see it being used anywhere

Reply:
It will be haha when I finish. It's going to be much more nice in the future. Plan is to use it to determine when to periodically save to UserDefaults because UserDefaults is really slow.